bson.cpp

```diff
@@ -386,8 +385,7 @@ void VariantToBsonConverter::_convertSerializable(bson_t *bson, const char *key,
 		result.asTypedValue(),
 		m,
 		v.get(),
-		nullptr,
-		1, args
+		nullptr
```

Comment:
What changed here? Were you needlessly passing the first argument (if it existed) on to `bsonSerialize()`?